spec/integration/test/test_helper_spec.rb

```diff
@@ -12,7 +12,11 @@
 
     Puppet::Util::Windows::Process.set_environment_variable(@varname, @rune_utf8)
   }
-
+  after(:all) {
+    @varname = 'test-helper-foo'

```

Comment:
I think the intent is to reuse `@varname` everywhere - i.e. this should be unnecessary, no?